utils/log.py

```diff
@@ -445,9 +445,14 @@ def slaveid(self):
         return SLAVEID or ""
 
     def art_log(self, level_name, message, kwargs):
+        if isinstance(message, unicode):
+            # Prevent UnicodeDecodeError
+            message = message.encode("ascii", "replace")
```

Comment:
Seems worthwhile to always encode it and skip the unicode check; python is smart enough to know what to do when encoding an ascii string as ascii.

Could we use xmlcharrefreplace instead of replace so we know what the character was supposed to be?